script.js

```diff
@@ -80,3 +80,40 @@ document.addEventListener('click', (e) => {
 });
 
 StorageB.display();
+
+// -------NAV--------
+
+const listBtn = document.getElementById('list-link');
+const addBtn = document.getElementById('add-link');
+const contactBtn = document.getElementById('contact-link');
+const listBox = document.querySelector('.list');
+const addBox = document.querySelector('.add');
+const contactBox = document.querySelector('.contact');
+
+listBtn.addEventListener('click', () => {
+  listBox.style.display = 'flex';
+  addBox.style.display = 'none';
+  contactBox.style.display = 'none';
+});
+
+addBtn.addEventListener('click', () => {
+  listBox.style.display = 'none';
+  addBox.style.display = 'flex';
+  contactBox.style.display = 'none';
+});
```

Comment:
- [ ] [OPTIONAL] Good job with your project, it works as expected :100:  Please ensure that when submitting the information, your form inputs are reset, because it will prevent, thinking that the data has not been submitted and avoid manually erasing before entering new information. :+1: